webapp/src/js/components/containers/TableWithLegend.js

```diff
@@ -35,30 +43,39 @@ let TableWithLegend = createReactClass({
   },
 
   render() {
-    let {table, query, order, columns, columnWidths, children} = this.props;
+    let {table, query, order, columns, children, classes} = this.props;
     order = this.state.order || order;
     query = this.getDefinedQuery(query, table);
 
+    // NOTE: Samples_and_Variants example is arriving as string
+    // <TableWithLegend table="samples" columns='["key", "Site_ID"]'>
```

Comment:
Is the prop arriving as string even when PropTypes.array is used?
The template system uses the PropType to convert the string to array/object/number